registry/spec.md

```diff
@@ -38,9 +41,20 @@ provide a common interaction pattern for these types of services with the goal
 of providing an interoperable framework that will enable common tooling and
 automation to be created.
 
-This specification itself is meant to be a framework from which additional
-specifications will be defined that expose model specific resources and
-metadata.
+The first section of this specification, ["Registry Formats and
+APIs"](#registry-formats-and-apis), is meant to be a framework from which
+additional specifications can be defined that expose model specific resources
+and metadata.
+
+The second section of this specification ["CloudEvents
```

Comment:
In prep for moving this to the xReg repo, it might be good to rename this something more generic